docs/src/pages/customization/css-in-js.md

```diff
@@ -155,6 +155,22 @@ function App() {
 export default App;
 ```
 
+## Plugins
+
+JSS uses the concept of plugins to extend his core.
+It's allowing people to cherry-pick the features they need.
+You pay the performance overhead for only what's you are using.
```

Comment:
JSS uses the concept of plugins to extend its core,
allowing you to cherry-pick the features you need,
and only paying the performance overhead for what you are using.